reclass/datatypes/tests/test_parameters.py

```diff
@@ -188,6 +188,18 @@ def test_merge_dicts_overwrite(self):
         goal.update(mergee)
         self.assertDictEqual(p.as_dict(), dict(dict=goal))
 
+    def test_merge_dicts_override(self):
```

Comment:
I was WAAAAY too tired when I wrote those. Replicated the functionality of other tests AND missed validating that we override properly on list extensions as well!

Much better and cleaner now.